src/test/java/data/structures/and/algorithms/BinarySearchTest.java

```diff
@@ -1,9 +1,31 @@
 package data.structures.and.algorithms;
 
+import org.junit.Test;
+
 import static org.junit.Assert.*;
 
 public class BinarySearchTest {
-
+BinarySearch obj = new BinarySearch();
```

Comment:
You made you method `static`, so you don't have to instantiate an object like this to call it.  Just do the name of the class, the dot operator, the name of the method: `BinarySearch.binarySearch( . . . )`